expression/builtin_string_vec_test.go

```diff
@@ -504,6 +504,16 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 			newRealStringGener(),
 			&numStrGener{*newRangeInt64Gener(-10, 40)},
 		}},
+		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETDecimal, types.ETInt, types.ETString}, geners: []dataGenerator{
+			newRangeDecimalGener(-10000, 10000, 0.5),
+			newRangeInt64Gener(-10, 40),
+			&constStrGener{"en_US"},
```

Comment:
can we write a local generator to test other values? include null

Reply:
OK. I'm not familar with `go test`, what's the minimal unit test command should I execute after fix these changes?

Reply:
> can we write a local generator to test other values? include null

currently only 'en_US' is implemented in `parser/mysql`, so i'm going to test null additionally. BTW, how to generate null 😭

Reply:
You can refer to the implementation of `timeFormatGener` in `expression/bench_test.go`